src/analyses/extractPthread.ml

```diff
@@ -146,12 +144,10 @@ module Tbls = struct
 
     let get k = Hashtbl.find table k
 
-    let get_key v = table |> Hashtbl.enum |> List.of_enum |> List.assoc_inv v
+    let get_key v = table |> Hashtbl.bindings |> List.assoc_inv v
```

Comment:
This looks like it could be similar to the `get_key` above to avoid constructing the intermediate list.